jsonschema-core.xml

```diff
@@ -365,6 +365,36 @@
                         and implementations.
                     </t>
                 </section>
+                <section title="Schema Vocabularies">
+                    <t>
+                        A schema vocabulary, or simply a vocabulary, is a set of keywords,
+                        their syntax, and their semantics.  A vocabulary is generally organized
+                        around a particular purpose.  Different uses of JSON Schema will
+                        involve different sets of vocabularies.
+                    </t>
+                    <t>
+                        Vocabularies are the primary unit of re-use in JSON Schema, as schema
+                        authors can indicate what vocabularies are required or optional in
+                        order to process the schema.  While keywords can be supported outside
+                        of any vocabulary, there is no analogous mechanism to indicate individual
```

Comment:
While arbitrary extra keywords are allowed in JSON Schema, vocabularies are how we can define their meaning in a portable way, which other tooling can then start to support. 

Or something.